content/en/admin/config.md

```diff
@@ -645,14 +645,6 @@ The password used to authenticate with sentinels.
 
 If set to `true`, Mastodon will use Elasticsearch for its search functions.
 
-### Elasticsearch {#elasticsearch}
-
-{{< page-ref page="admin/elasticsearch" >}}
-
-#### `ES_ENABLED`
-
-If set to `true`, Mastodon will use Elasticsearch for its search functions.
-
```

Comment:
Is that removal intended?

Reply:
It is an unrelated change that I noticed, this section was duplicated